tests/neg/arg-eof.scala

```diff
@@ -0,0 +1,3 @@
+object Test:
+  case class Widget(name: String, other: Int = 5)
+  Widget(name = "foo", // error // error
```

Comment:
Why the double `error` ?

Reply:
The second one is a `Required: Int\nFound: Null` error. It's fixed by https://github.com/lampepfl/dotty/pull/14492.

Reply:
Ach! cool!